Automattic/wp-calypso
# Hosting: Answer common questions for SFTP/DB inline to the card, also add missing analytics

fixes https://github.com/Automattic/wp-calypso/issues/37890

This PR updates SFTP/DB to answer top questions, also updates the subheader to "Access and manage more advanced settings of your website." Most copy was taken from @automattic-ian's new support docs 💖.  

I've also updated the SFTP card to have a copy button for each field, but I'm sure we may want to style this a bit more:

<img width="756" alt="Screen Shot 2019-11-22 at 5 19 07 PM" src="https://user-images.githubusercontent.com/1270189/69470635-b6933e00-0d4c-11ea-925e-3af6ea5fde0a.png">


![accordian](https://user-images.githubusercontent.com/1270189/69465832-1bdc3480-0d37-11ea-854e-fa6a85fd78fe.gif)


| Original  | SFTP Not Enabled | SFTP Enabled | Small Viewport |
| ------------- | ------------- |------------- |------------- |
| <img width="1561" alt="Screen Shot 2019-11-22 at 2 44 56 PM" src="https://user-images.githubusercontent.com/1270189/69465717-bf791500-0d36-11ea-8fe8-f8eeacede0a7.png"> | <img width="1550" alt="Screen Shot 2019-11-22 at 2 39 18 PM" src="https://user-images.githubusercontent.com/1270189/69465731-c99b1380-0d36-11ea-9095-bc465a9a1e23.png"> |  <img width="1416" alt="Screen Shot 2019-11-22 at 2 45 49 PM" src="https://user-images.githubusercontent.com/1270189/69465756-df103d80-0d36-11ea-9775-6b9f82c32e7e.png"> | <img width="494" alt="Screen Shot 2019-11-22 at 2 50 05 PM" src="https://user-images.githubusercontent.com/1270189/69465937-81302580-0d37-11ea-80ea-3b6f163ebae0.png">|

I've also opened a related issue https://github.com/Automattic/wp-calypso/issues/37899

### Testing Instructions
- Create a new site, with a business plan. Transfer to atomic by installing a plugin.
- Visit http://calypso.localhost:3000/hosting-config/
- Verify that styling copy make sense
- Verify that we can still enable SFTP credentials and DB access

#### Analytics
- In console set `localStorage.debug='calypso:analytics*';`
- Clicking on "Contact Us" triggers the following analytics events:
<img width="1118" alt="Screen Shot 2019-11-22 at 3 36 03 PM" src="https://user-images.githubusercontent.com/1270189/69467705-146c5980-0d3e-11ea-8218-bd17b9340c19.png">
- Clicking on "Reset password" in the SFTP card triggers the following analytics events
<img width="1152" alt="Screen Shot 2019-11-22 at 3 47 19 PM" src="https://user-images.githubusercontent.com/1270189/69468117-c9534600-0d3f-11ea-9531-c0701f7968e1.png">
- Clicking on "Open phpMyAdmin" in the Database access card triggers the following events:
<img width="1183" alt="Screen Shot 2019-11-22 at 4 02 14 PM" src="https://user-images.githubusercontent.com/1270189/69468736-31a32700-0d42-11ea-864c-e9895355f200.png">
- Clicking on "Restore" in the dialog of the Database Access card triggers the following events
<img width="1131" alt="Screen Shot 2019-11-22 at 4 12 18 PM" src="https://user-images.githubusercontent.com/1270189/69468950-30bec500-0d43-11ea-986a-0d107b99100e.png">
- Clicking on Enable SFTP Credentials triggers the following events:
<img width="981" alt="Screen Shot 2019-11-22 at 4 17 14 PM" src="https://user-images.githubusercontent.com/1270189/69469327-0b32bb00-0d45-11ea-8e67-ffb6796d663a.png">
